libweb3jsonrpc/JsonHelper.cpp

```diff
@@ -360,8 +360,8 @@ TransactionSkeleton toTransactionSkeleton(Json::Value const& _json)
 
 	if (!_json["from"].empty())
 		ret.from = jsToAddress(_json["from"].asString());
-	if (!_json["to"].empty() && _json["to"].asString() != "0x")
-		ret.to = jsToAddress(_json["to"].asString());
+    if (!_json["to"].empty() && _json["to"].asString() != "0x" && !_json["to"].asString().empty())
```

Comment:
This seems to be a bugfix?

Reply:
yes. because empty() is true if there is no "to" json section. 
but the string in this section could be empty or 0x (looks like no consensus on format here)

Reply:
Should there be a PR for this bugfix?

Reply:
Ping 😉

Reply:
ask/bid price PR is yet not merged.

Reply:
How is that relevant?

Reply:
ok. I have now finished fixing the tests and could concentrate on PRs here.

Reply:
@axic  https://github.com/ethereum/cpp-ethereum/pull/4923

Reply:
Thanks!